.github/workflows/mayhem.yml

```diff
@@ -79,8 +79,15 @@ jobs:
     - name: Build and push the app images
       run: |
         docker compose pull --ignore-buildable -q
+        
+        # Push with commit sha
         SHA=${{ github.sha }} docker compose build
         SHA=${{ github.sha }} docker compose push -q
+        
+        # Push with latest
+        docker compose build
```

Comment:
I think you can use `docker image tag` to avoid rebuilding - maybe not totally necessary but probably more clear

Reply:
Since the build is already cached it hardly takes any time :)

Reply:
(beyond what's in the file already)